### ISSUE TITLE
Storm of INFO REPLICATION commands causing stability issues in AWS Elasticache for REDIS

### ISSUE THREAD

**Issue by pthargis** [StackExchange/StackExchange.Redis#2188]
Hello, 

We are running with the 2.2.50 version of StackExchange.Redis Library. Working with AWS, we found out that INFO REPLICATION commands are storming AWS Elasticache for REDIS and causing REDIS to go down. Any help would be greatly appreciated? We are using Cluster Mode Enabled (multiple Shards) and TLS

**Comment by pthargis** [comment]
@deepakverma @NickCraver - You have worked on this kind of issue in the past. can you help?

**Comment by NickCraver** [comment]
We've had several fixes around these scenarios after 2.2.50 - can you repro on the latest release?

**Comment by pthargis** [comment]
will update you. Thank you

**Comment by pthargis** [comment]
@NickCraver - Version upgrades on our side are going to take a significant amount of time. We have started the work. Is there a workaround available to temporarily stop these storms? Is there a way to exclude INFO Commands or INFO Replication Commands? Is there a way that to limit the library to stop sending these commands?

**Comment by NickCraver** [comment]
I don't believe there's a way to limit these currently, without breaking the library - we depend on it to see what the state of servers we're connecting to is. If you're on an older version of the library, there's an even greater chance of breaks (we solve _some_ issues later, but I don't recall `INFO` changing any)

**Comment by pthargis** [comment]
Thank you @NickCraver. we thought of using the command map option to exclude these commands. https://stackexchange.github.io/StackExchange.Redis/Configuration#automatic-and-manual-configuration 

It won't help. correct?

**Comment by pthargis** [comment]
Ack. Thanks

**Comment by pthargis** [comment]
Updated the original post with note saying that TLS is enabled

**Comment by NickCraver** [comment]
@pthargis checking in here - how are y'all getting along with this, any update?

**Comment by pthargis** [comment]
@NickCraver - We haven't upgraded the library yet. We reduced the number of ConnectionMultiplexer that we create. We will be upgrading to the latest version in next few months
